src/react/account/details/properties/AccountKeys.jsx

```diff
@@ -0,0 +1,189 @@
+                    <EllipsisCell title={access_key}>{access_key}</EllipsisCell>
+                    <Clipboard text={access_key}/>
+                </span>;
+            },
+        },
+        {
+            Header: 'Created On',
+            accessor: 'created_at',
+            Cell({ value: created_at }: { value: Date }) { return formatDate(new Date(created_at)); },
+        },
+        {
+            Header: 'Last Used',
+            accessor: 'last_used',
+            Cell({ value: last_used }: { value: Date }) { return formatDate(new Date(last_used)); },
+        },
+        {
+            id: 'actions',
+            Header: '',
+            accessor: 'access_key',
+            disableSortBy: true,
+            Cell({ value: access_key }: { value: string }){
+                return (
+                    <T.Actions>
+                        <Tooltip overlay="Remove Key" placement="left">
+                            <T.ActionButton
+                                title="Remove Key"
+                                disabled={false}
+                                icon={<i className="fas fa-trash" />}
+                                onClick={() => dispatch(deleteAccessKey(access_key, account.userName))}
+                                size="smaller"
+                                variant="danger"
+                                text=''
+                            />
+                        </Tooltip>
+                    </T.Actions>
+                );
+            },
+        },
+    ], [account.userName, dispatch]);
+
+    const accessKeys = useMemo(() => accessKeysInfo.map(accessKeyInfo => {
+        return {
+            access_key: accessKeyInfo.AccessKeyId,
+            created_at: accessKeyInfo.CreateDate,
+            last_used: accessKeyInfo.LastUsed,
+        };
+    }), [accessKeysInfo]);
+
+    const {
+        getTableProps,
+        getTableBodyProps,
+        headerGroups,
+        rows,
+        prepareRow,
+    } = useTable({
+        columns,
+        data: accessKeys,
+        initialState: { sortBy: initialSortBy },
+        disableSortRemove: true,
+        autoResetFilters: false,
+        autoResetSortBy: false,
+    }, useFilters, useSortBy);
+
+    const displaySecurityStatus = accessKeys && accessKeys.length >= 1;
+
+    return (
+        <TableContainer>
+            <h3>Root user Access Key & Secret Keys details</h3>
+            <div style={{ display: 'flex', justifyContent: displaySecurityStatus ? 'space-between' : 'flex-end' }}>
+                { displaySecurityStatus &&
+                    <Banner
+                        variant="danger"
+                        icon={<i className="fas fa-exclamation-triangle" />}
+                    >
+                        Security Status: delete your Access keys (as a best practice, create an IAM user with access keys).
+                    </Banner>
+                }
+                <Button
+                    icon={<i className="fas fa-plus" />}
```

Comment:
To create account access key, use `API.generateKeyConfigurationOverlayUser()`
You will find its definition here: http://management.zenko.local/swagger.json

Reply:
This ticket don't ask to add the access key creation, so I remove this button "Create Access Key".

From jira:
> The creation form when clicking on the create button will be in a second iteration.